novawallet/Modules/Staking/StartStakingInfo/ParachainStaking/StartStakingInfoParachainPresenter.swift

```diff
@@ -0,0 +1,161 @@
+import SoraFoundation
+import BigInt
+
+final class StartStakingInfoParachainPresenter: StartStakingInfoBasePresenter {
+    let interactor: StartStakingInfoParachainInteractorInputProtocol
+
+    private var state: State = .init() {
+        didSet {
+            if state != oldValue {
+                provideViewModel(state: state)
+            }
+        }
+    }
+
+    init(
+        interactor: StartStakingInfoParachainInteractorInputProtocol,
+        wireframe: StartStakingInfoWireframeProtocol,
+        startStakingViewModelFactory: StartStakingViewModelFactoryProtocol,
+        localizationManager: LocalizationManagerProtocol,
+        applicationConfig: ApplicationConfigProtocol
+    ) {
+        self.interactor = interactor
+
+        super.init(
+            interactor: interactor,
+            wireframe: wireframe,
+            startStakingViewModelFactory: startStakingViewModelFactory,
+            localizationManager: localizationManager,
+            applicationConfig: applicationConfig
+        )
+    }
+
+    override func setup() {
+        super.setup()
+        view?.didReceive(viewModel: .loading)
+    }
+}
+
+extension StartStakingInfoParachainPresenter: StartStakingInfoParachainInteractorOutputProtocol {
+    func didReceive(networkInfo: ParachainStaking.NetworkInfo?) {
+        state.networkInfo = networkInfo
+    }
+
+    func didReceive(error: ParachainStartStakingInfoError) {
+        switch error {
```

Comment:
add error logging here. This will allow fast error detection when application is running in debug